src/System.Web.HttpUtility/tests/project.json

```diff
@@ -0,0 +1,31 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore.Platforms": "4.3.0-beta-devapi-24513-01",
+    "System.Collections.Specialized": "4.3.0-beta-devapi-24513-01",
+    "System.Diagnostics.Tracing": "4.3.0-beta-devapi-24513-01",
+    "System.Linq.Expressions": "4.3.0-beta-devapi-24513-01",
+    "System.ObjectModel": "4.3.0-beta-devapi-24513-01",
+    "System.Resources.ResourceManager": "4.3.0-beta-devapi-24513-01",
+    "System.Runtime": "4.3.0-beta-devapi-24513-01",
+    "System.Runtime.Extensions": "4.3.0-beta-devapi-24513-01",
+    "System.Text.RegularExpressions": "4.3.0-beta-devapi-24513-01",
+    "System.Threading": "4.3.0-beta-devapi-24513-01",
+    "test-runtime": {
+      "target": "project",
+      "exclude": "compile"
+    },
+    "Microsoft.xunit.netcore.extensions": "1.0.0-prerelease-00807-03",
+    "Microsoft.DotNet.BuildTools.TestSuite": "1.0.0-prerelease-00807-03"
+  },
+  "frameworks": {
+    "netstandard1.3": {}
+  },
+  "supports": {
+    "coreFx.Test.netcoreapp1.0": {},
+    "coreFx.Test.netcore50": {},
+    "coreFx.Test.net46": {},
+    "coreFx.Test.net461": {},
```

Comment:
@karajas defined these as independent (not cumulative), so you need each one if you want to run tests there.